Dockerfile

```diff
@@ -17,13 +17,13 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 ARG BUILD_SRC="/usr/local/src"
 
 ARG JANUS_VERSION="v1.0.0"
-ARG JANUS_LIBNICE_VERSION="0.1.17"
+ARG JANUS_LIBNICE_VERSION="0.1.18"
 ARG JANUS_PAHO_MQTT_VERSION="v1.3.4"
-ARG JANUS_RABBITMQ_VERSION="v0.10.0"
+ARG JANUS_RABBITMQ_VERSION="v1.0.4"
 ARG JANUS_LIBSRTP_VERSION="2.3.0"
-ARG JANUS_USRSCTP_VERSION="547d3b46c64876c0336b9eef297fda58dbe1adaf"
-ARG JANUS_BORINGSSL_VERSION="8b601c88fb9b82b49407b018ff5e626a59600669"
-ARG JANUS_LIBWEBSOCKETS_VERSION="v4.0.20"
+ARG JANUS_USRSCTP_VERSION="0.9.5.0"
+ARG JANUS_BORINGSSL_VERSION="ac3f4fb8e1e3fd6da51033514e247e7924b16439"
```

Comment:
Checking https://boringssl.googlesource.com/boringssl/+log/refs/heads/master we see that there are newer commits.
Why not use `225e8d39b50757af56e61cd0aa7958c56c487d54`?

Reply:
Basically, I'm not taking commits after the Janus `v1.0.0` release date... But I can try latest versions as well.